templates/macros.xml.j2

```diff
@@ -2,8 +2,8 @@
 <!-- {{ ansible_managed }} -->
 <yandex>
     <macros>
-        <cluster>replication_test_1shards_2replicas</cluster>
+        <cluster>{{ clickhouse_display_name | default('clickhouse') }}</cluster>
         <shard>01</shard>
```

Comment:
Should the shard value be customizable as well ?

Reply:
Thanks for the feedback.
Yes I think we should add it later for sure when we will want to activate sharding, for now all tables even replicated will have only one shard.